Lib/test/test_zstd/test_core.py

```diff
@@ -0,0 +1,2506 @@
+import array
+import gc
+import io
+import pathlib
+import pickle
+import random
+import builtins
+import re
+import os
+import unittest
+import tempfile
+import threading
+
+from compression._common import _streams
+
+from test.support.import_helper import import_module
+from test.support import threading_helper
+from test.support import _1M
+from test.support import Py_GIL_DISABLED
+
+zstd = import_module("compression.zstd")
+_zstd = import_module("_zstd")
+from compression.zstd import (
+    zstdfile,
+    compress,
+    decompress,
+    ZstdCompressor,
+    ZstdDecompressor,
+    ZstdDict,
+    ZstdError,
+    zstd_version,
+    zstd_version_info,
+    compressionLevel_values,
+    get_frame_info,
+    get_frame_size,
+    finalize_dict,
+    train_dict,
+    CParameter,
+    DParameter,
+    Strategy,
+    ZstdFile,
+    zstd_support_multithread,
+)
+from compression.zstd.zstdfile import open
+
+_1K = 1024
+_130_1K = 130 * _1K
+DICT_SIZE1 = 3*_1K
+
+DAT_130K_D = None
+DAT_130K_C = None
+
+DECOMPRESSED_DAT = None
+COMPRESSED_DAT = None
+
+DECOMPRESSED_100_PLUS_32KB = None
+COMPRESSED_100_PLUS_32KB = None
+
+SKIPPABLE_FRAME = None
+
+THIS_FILE_BYTES = None
+THIS_FILE_STR = None
+COMPRESSED_THIS_FILE = None
+
+COMPRESSED_BOGUS = None
+
+SAMPLES = None
+
+TRAINED_DICT = None
+
+KB = 1024
+MB = 1024*1024
```

Comment:
I see you tried to use `_KB` and `_MB` in tests already, maybe these line can be removed?